### PR TITLE
Dev aadi quiz app

### PR DESCRIPTION
**Pull request by aadijain725**
The completed initial version of quiz component.

### CODE REVIEW

File: algowiz-app/src/App.css [hunk 1]
```diff
@@ -36,3 +36,19 @@
     transform: rotate(360deg);
   }
 }
+
+#quiz-submit-btn {
+  width: 100%;
+  margin-top: 5%;
+}
+
+
```
Comment (MadEste) [inline algowiz-app/src/App.css:45]: 👍

**Review by MadEste — APPROVED** [review]
This all looks good to me

**Comment by MadEste** [comment]
@shivamdb Can you fix the tests so we can merge (json in the test file is not working)

**Review by shivamdb — APPROVED** [review]
Looks good

**Pull request merged**